fabulaws/ubuntu/instances/base.py

```diff
@@ -76,6 +77,32 @@ def _encrypt_device(self, device, passwd=None):
         return device
 
     @uses_fabric
+    def _mount_and_persist(self, device, mount_point, fs_type=None, opts=None):
+        mount_cmd = ' '.join([
+            'mount',
+            '-o {}'.format(opts) if opts else '',
+            '-t {}'.format(fs_type) if fs_type else '',
+            device,
+            mount_point,
+        ])
+        fstab_entry = ' '.join([
+            device,
+            mount_point,
+            fs_type if fs_type else self.fs_type,
+            opts if opts else 'defaults',
+            '0',
+            '0',
+        ])
+        sudo(mount_cmd)
+        if self.fs_encrypt:
+            if not files.exists(self.mount_script):
+                files.append(self.mount_script, "#!/bin/sh", use_sudo=True)
```

Comment:
The only reason for that is so that we can add the shebang line as the first in the file.